.github/workflows/android_automatic_pull_request_review.yml

```diff
@@ -0,0 +1,50 @@
+name: Android - Automatic pull request review
+
+on:
+  pull_request:
+    types: [ opened, reopened, edited, synchronize ]
+
+jobs:
+  review_android_pull_request:
+    name: Pull request review for Android
+    runs-on: macos-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout source code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Android Signing
+        env:
+          ANDROID_RELEASE_KEYSTORE: ${{ secrets.ANDROID_RELEASE_KEYSTORE }}
+          ANDROID_SIGNING_PROPERTIES: ${{ secrets.ANDROID_SIGNING_PROPERTIES }}
+        run: |
+          touch config/release.keystore
+          echo $ANDROID_RELEASE_KEYSTORE | base64 --decode > config/release.keystore
+          touch signing.properties
+          echo $ANDROID_SIGNING_PROPERTIES | base64 --decode > signing.properties
+
+      - name: Setup Java JDK
+        uses: actions/setup-java@v2.1.0
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+
+      - name: Cache Gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Build and Test
+        run: ./gradlew allTests
```

Comment:
There is no Lint and Detekt checks, please plan for that https://github.com/nimblehq/android-templates/blob/develop/.github/workflows/review_pull_request.yml#L32-L34

Reply:
Yest, I already mentioned about detekt on this PR: https://github.com/suho/kmm-ic/pull/75

Technically, because I'm using Jetpack Compose, the rules might different with current rules from android-templates, so I did not put detekt into this project. However, I think I will add the default rules :D